ReactWindows/ReactNative.Shared/Views/TextInput/ReactTextBox.cs

```diff
@@ -151,5 +229,10 @@ private void OnSelectionChanged(object sender, RoutedEventArgs e)
                         start,
                         start + length));
         }
+
+        private bool IsBeingEdited()
+        {
+            return !(FocusState == FocusState.Unfocused);
```

Comment:
For WPF apparently you need ifdef here since the method to check is called IsKeyboardFocused()

Reply:
added